client/src/components/App.jsx

```diff
@@ -79,35 +80,51 @@ class App extends React.Component {
     });
     this.getLocation();
   }
-
+  // grab location of user
   getLocation() {
     console.log('getting location');
+    let correctThis = this;
```

Comment:
I'm like... 75% sure that if you changed your function on line 90 to an arrow function, you could use `this` as intended without having to manually set it to a variable. Something to experiment with!

Reply:
Wow I actually did not think about that. It makes sense though so I will check it out! Thanks